anchor_positions:
  "388": [0.0, 0.0, 0.94]
  "644": [0.0, 1.205, 1.75]
  "900": [1.683, 1.205, 2.06]
  "1156": [1.683, 0.0, 1.33]

